python/paddle/fluid/incubate/fleet/collective/__init__.py

```diff
@@ -42,6 +46,21 @@ def __init__(self):
         pass
 
 
+class TrainStatus(object):
```

Comment:
Is it necessary construct a persistable training status object to recording context like training steps, the offset of filelist?

Reply:
They can be added when the action is implemented.